lib/internal/v8_prof_polyfill.js

```diff
@@ -26,40 +26,40 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 // Node polyfill
-var fs = require('fs');
-var os = {
+const fs = require('fs');
+const cp = require('child_process');
+const os = {
   system: function(name, args) {
     if (process.platform === 'linux' && name === 'nm') {
       // Filter out vdso and vsyscall entries.
-      var arg = args[args.length - 1];
+      const arg = args[args.length - 1];
       if (arg === '[vdso]' ||
           arg == '[vsyscall]' ||
           /^[0-9a-f]+-[0-9a-f]+$/.test(arg)) {
         return '';
       }
     }
-    return require('child_process').execFileSync(
-        name, args, {encoding: 'utf8'});
+    const cmd = name + ' ' + args.join(' ');
+    return cp.execSync(cmd, {encoding: 'utf8'});
```

Comment:
On Windows there is no `/bin/sh`, you'd use `cmd.exe /s /c`, but I don't think you actually need to spawn a shell.  The OS X case is special because you're executing shell script, on other platforms `nm` is invoked directly.